output.go

```diff
@@ -22,9 +22,18 @@ type Glyph struct {
 	YAdvance fixed.Int26_6
 	XOffset  fixed.Int26_6
 	YOffset  fixed.Int26_6
-	Cluster  int
-	Glyph    fonts.GID
-	Mask     harfbuzz.GlyphMask
+	// Cluster is the lowest rune index of all runes shaped into
+	// this glyph cluster. All glyphs sharing the same cluster value
+	// are part of the same cluster and will have identical RuneCount
+	// and GlyphCount fields.
+	Cluster int
```

Comment:
Just reading through this I stumbled a little. Is this a `ClusterID`?
Indeed, should `Glyph` be `GlyphID` for similar reasons? It is not really a glyph in the fullest definition is it?

Reply:
`Cluster` is really a standard index into the `Text` slice input. For clarity, I would rather rename it to `ClusterIndex`.
`Glyph` is a bit different : it has no signification outside a font file context. So we could indeed rename `Glyph` to `GlyphID` (even though the `fonts.GID` type is somewhat already documenting the purpose of this field).